ansible/playbooks/roles/repository/tasks/copy-download-requirements.yml

```diff
@@ -55,9 +55,16 @@
         dest: "{{ download_requirements_dir }}/{{ item }}"
       loop: "{{ _files }}"
 
+    - name: Filter sensitive data from the manifest
+      filter_credentials:
+        src: "{{ input_manifest_path }}"
+        dest: /tmp/filtered_manifest.yml
+      delegate_to: localhost
+      when: not full_download and input_manifest_path
```

Comment:
The next task has the same condition so I would suggest to use `block` to have just one `when`.